server/src/tenant-manager/tenant-manager.service.ts

```diff
@@ -126,6 +138,59 @@ export class TenantManagerService {
     standardObjectsPrefillData(workspaceDataSource, dataSourceMetadata.schema);
   }
 
+  public async injectWorkspaceData(
+    table: string,
+    workspaceId: string,
+    values,
+    columns,
+  ) {
+    await this.init(workspaceId, false);
+    const schema = this.tenantDataSourceService.getSchemaName(workspaceId);
+    const workspaceDataSource =
+      await this.tenantDataSourceService.connectToSchemaDataSource(schema);
+
+    await workspaceDataSource.transaction(
```

Comment:
Could we use chunk here too?

Reply:
we can but it is not necessary for the actual production database